product_docs/docs/pgd/5/reference/routing.mdx

```diff
@@ -7,7 +7,7 @@ rootisheading: false
 
 ### `bdr.create_proxy`
 
-Create a proxy configuration
+Create a proxy configuration.
```

Comment:
Can you just the single quotes vs. back ticks in this file? It seemed to use both, but I didn't want to "correct" single quotes to back ticks if that wasn't intended. I'd be happy to change them all to back ticks if that's correct.

Reply:
"just the"? adjust the?